code/extensions/che-api/extension.webpack.config.js

```diff
@@ -16,20 +16,34 @@
 
 const withDefaults = require('../shared.webpack.config');
 const webpack = require('webpack');
+const { merge } = require('webpack-merge');
 
-module.exports = withDefaults({
-	context: __dirname,
-	resolve: {
-		mainFields: ['module', 'main']
-	},
-	entry: {
-		extension: './src/extension.ts',
-	},
-	externals: {
-		'bufferutil': 'commonjs bufferutil', // ignored
-		'utf-8-validate': 'commonjs utf-8-validate', // ignored
-	},
-	plugins: [
-		new webpack.ContextReplacementPlugin(/keyv/), // needs to exclude the package to ignore warnings https://github.com/jaredwray/keyv/issues/45
-	],
+const config = withDefaults({
+    context: __dirname,
+    resolve: {
+        mainFields: ['module', 'main'],
+    },
+    entry: {
+        extension: './src/extension.ts',
+    },
+    externals: {
+        'bufferutil': 'commonjs bufferutil', // ignored
+        'utf-8-validate': 'commonjs utf-8-validate', // ignored
+    },
+    plugins: [
+        new webpack.ContextReplacementPlugin(/keyv/), // needs to exclude the package to ignore warnings https://github.com/jaredwray/keyv/issues/45
+    ],
 });
+
+module.exports = merge(config, {
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/,
+                resolve: {
+                    fullySpecified: false, // This avoids the issue with the devfile/api extension requirement
```

Comment:
Could you add a bit more comments? Or just attach a link to the issue describing the problem.

Reply:
without these changes we have errors from Webpack like:
```
#13 2791.9 [19:40:58] Error in plugin "webpack-stream"
#13 2791.9 Message:
#13 2791.9     Module not found: Error: Can't resolve './http/http' in '/checode-compilation/extensions/che-github-authentication/node_modules/@devfile/api/dist'
#13 2791.9 Did you mean 'http.js'?
#13 2791.9 BREAKING CHANGE: The request './http/http' failed to resolve only because it was resolved as fully specified
#13 2791.9 (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
```